Lang/French.pj.Lang

```diff
@@ -90,94 +90,94 @@
+#267#  "Basculer l’émulation du mode fenêtré vers le mode plein écran."
 #268#  "Permet de mettre la fenêtre Project64 au premier plan"
 #269#  "Configurer le plugin graphique"
 #270#  "Configurer le plugin audio"
 #271#  "Configurer le plugin contrôleur (ex : choisir les touches)"
 #272#  "Configurer le plugin RSP"
-#273#  "Afficher l'utilisation du CPU de l'émulateur répartie sur les différentes tâches"
+#273#  "Afficher l’utilisation du CPU de l’émulateur répartie sur les différentes tâches"
 #274#  "Voir/changer les paramètres pour cette application"
-#275#  "Voir le manuel pour l'application"
-#276#  "Voir la FAQ pour l'application"
-#278#  "A Propos de l'application et des auteurs"
-#277#  "A Propos des auteurs sur les fichiers de support"
-#279#  "Ouvrir l'image ROM ouverte précédemment"
+#275#  "Voir le manuel pour l’application"
+#276#  "Voir la FAQ pour l’application"
+#278#  "À propos de l’application et des auteurs"
+#277#  "À propos des auteurs sur les fichiers de support"
+#279#  "Ouvrir l’image ROM ouverte précédemment"
 #280#  "Choisir ce répertoire comme votre répertoire de jeu"
-#281#  "Changer l'application pour utiliser cette langue"
-#282#  "Choisir l'emplacement de sauvegarde pour l'état sauvegardé"
+#281#  "Changer l’application pour utiliser cette langue"
+#282#  "Choisir l’emplacement de sauvegarde pour l’état sauvegardé"
 #283#  "Jouer au jeu sélectionné"
-#284#  "Information concernant le jeu sélectionné"
-#285#  "Editer les paramètres pour le jeu sélectionné"
-#286#  "Editer les cheats pour le jeu sélectionné"
+#284#  "Informations concernant le jeu sélectionné"
+#285#  "Éditer les paramètres pour le jeu sélectionné"
+#286#  "Éditer les cheats pour le jeu sélectionné"
 
 /*********************************************************************************
 * ROM Browser                                                                    *
 *********************************************************************************/
 //ROM Browser Fields
-#300#  "Nom du Fichier"
-#301#  "Nom Interne"
-#302#  "Nom Correcte (GOODN64)"
+#300#  "Nom du fichier"
+#301#  "Nom interne"
+#302#  "Nom correct (GoodN64)"
 #303#  "Statut"
 #304#  "Taille de la ROM"
 #305#  "Notes (noyau)"
 #306#  "Notes (plugins par défaut)"
 #307#  "Notes (utilisateur)"
-#308#  "ID de la Cartouche"
+#308#  "ID de la cartouche"
 #309#  "Constructeur"
 #310#  "Pays"
 #311#  "Développeur"
 #312#  "CRC1"
 #313#  "CRC2"
 #314#  "Puce CIC"
-#315#  "Date de Sortie"
+#315#  "Date de sortie"
 #316#  "Genre"
 #317#  "Joueurs"
-#318#  "Forcer le Feedback"
-#319#  "Format de Fichier"
+#318#  "Forcer le feedback"
+#319#  "Format de fichier"
 
 //Select ROM
-#320#  "Sélectionner le Répertoire ROM en cours"
+#320#  "Sélectionner le répertoire de ROM actuel"
 
 //Messages
-#340#  "Mauvaise ROM ? Utilisez GoodN64 & vérifier la mise à jour de RDB"
+#340#  "Mauvaise ROM ? Utilisez GoodN64 et vérifiez la mise à jour de RDB"

```

Comment:
I think "Mauvaise ROM ? Utilisez GoodN64 et vérifiez que le fichier RDB est à jour" is better here.